src/app/workflow/snapshot-exporter-modal/snaphot-exporter-modal.component.scss

```diff
@@ -0,0 +1,37 @@
+@import 'materialColorScheme.scss';
+
+.confirmation-box {
+  margin: 0 0 9px;
+  padding: 8px 0.3px 8px 10.7px;
+  border-radius: 8px;
+  border: solid 1px #fff176;
+  background-color: #fffde7;
+}
+
+.confirmation-text {
+  font-size: 12px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.33;
+  letter-spacing: normal;
+  color: mat-color($dockstore-app-gray, darken);
+}
+
+.account-link {
+  padding: 5px 15.3px 12px 10.7px;
+  border-radius: 8px;
+  border: solid 1px mat-color($dockstore-app-warn, 4);
+  background-color: mat-color($dockstore-app-warn, 5);
+}
+
+.link-action {
+  margin: 0 0 0 4px;
+  font-size: 12px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #3f51b5;
```

Comment:
if this color is going to be used alot, add it to palette

Reply:
Went with a variable for now; it's in the home, about, etc. reworked pages, but there doesn't seem to be consistent way it's used.